src/Symfony/Component/Security/Http/Firewall/AccessListener.php

```diff
@@ -24,23 +27,86 @@
  */
 class AccessListener implements ListenerInterface
 {
+    /**
+     * @var TokenStorageInterface
+     * @deprecated
+     */
+    private $tokenStorage;
```

Comment:
The docblocks and deprecation notices are not required for private properties (if the type is known). Private means it's not an extension point, thus don't have to marked as such.

Reply:
removed docblock. thanks.